src/git.rs

```diff
@@ -67,8 +65,19 @@ fn path_command(path: impl AsRef<Path>, subcommand: &[&str]) -> Option<PathBuf>
             .and_then(std::char::from_u32),
         Some('\n')
     );
-    let toplevel = OsString::from_vec(output.stdout);
-    Some(toplevel.into())
+    Some(path_from_output(output.stdout))
+}
+
+#[cfg(unix)]
+fn path_from_output(output: Vec<u8>) -> PathBuf {
+    use std::ffi::OsString;
+    use std::os::unix::prelude::OsStringExt;
+    OsString::from_vec(output).into()
+}
+
+#[cfg(all(not(unix)))]
```

Comment:
Drop the all().

Reply:
Does `gir` already use `cfg_if!`? Might have been nice to use within this function so that you don't have to repeat the function body nor negate the cfg clause, at the expense of two extra indent levels :/

Reply:
It does not. Should I add it ?